cms/.env.example

```
@@ -0,0 +1,13 @@
+# config/master.key の内容
+RAILS_MASTER_KEY=
+
+# 管理画面用のBasic認証
+BASIC_AUTH_USER=
+BASIC_AUTH_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BASIC_AUTH_PASSWORD key should go before the BASIC_AUTH_USER key

(UnorderedKey)